### PR TITLE
feat(common): rename setValue to setContent

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

As we decided earlier, this PR renames `setValue` to `setContent` which makes more sense.

## Why?

JIRA: EXT-1497

### CODE REVIEW

File: README.md
Comment (johannes-lindgren) [inline README.md:1]: See line 195

https://github.com/storyblok/field-plugin/pull/177/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L195

File: README.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ Properties:
 
 | Key                | Description                                                                                                                                                                                           |
 |--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `setValue`         | Updates the value of the field plugin. For example, `setValue(3.14159)`                                                                                                                               |
+| `setContent`         | Updates the value of the field plugin. For example, `setContent(3.14159)`                                                                                                                               |
```
Comment (johannes-lindgren) [inline README.md:213]:
```suggestion
| `setContent`         | Updates the content of the field plugin. For example, `setContent(3.14159)`                                                                                                                               |
```

**Review by johannes-lindgren — CHANGES_REQUESTED** [review]
One small change in the readme. 

We need to update the js template. Since it's not using typescript, it doesn't show an error when you build it...

Otherwise 👍

**Comment by eunjae-lee** [comment]
@johannes-lindgren I forgot to mention. Yes, I didn't update the JS template yet, because #176 is still open :) I will update it afterwards.

**Review by johannes-lindgren — APPROVED** [review]

**Pull request merged**